### ISSUE TITLE
License is installed to /usr

### ISSUE THREAD

**Issue by 0x9fff00** [viniciuschiele/flask-apscheduler#120]
Commit dadd5a42ef87286014be90ff080c187d9744e711, suggested by #57, added `LICENSE` to `data_files`. It appears that this was intended to add the license to the source distribution, however it also causes it to be installed in `/usr` when `python setup.py install` is run. This is incorrect as `/usr` should only contain the subdirectories specified in the Filesystem Hierarchy Standard, and no files.

**Comment by viniciuschiele** [comment]
Thanks for pointing this out, would you know what is the correct way to include extra files like LICENSE, CHANGELOGS in a pypi package?

**Comment by guru-florida** [comment]
I am getting this as well. The "data_files" property is for files you want to put under the install prefix (/usr ex) like icons, desktop shortcuts, etc. For this LICENSE file and other files that should be installed in the package dir it should use "package_data" property and set the include_package_data=True flag.

Lots of related discussion here:
https://stackoverflow.com/questions/11235820/setup-py-not-installing-data-files

Example:
```python
setup(
   name='myapp',
   packages=find_packages(),
   package_data={
      'myapp': ['data/*.txt'],
   },
   include_package_data=True,
   zip_safe=False,
   install_requires=['distribute'],
)
```
package_data is specifically for package releases, which I think is what we want here. More details:
https://stackoverflow.com/questions/7522250/how-to-include-package-data-with-setuptools-distutils

**Comment by viniciuschiele** [comment]
I released a new version 1.12.3, thank you all
